devices/mercator.js

```diff
@@ -184,6 +184,13 @@ module.exports = [
             await reporting.onOff(endpoint);
             await reporting.fanMode(endpoint);
         },
+        onEvent: async (type, data, device) => {
+            if (type === 'deviceJoined') {
+                // Device ships with {fanModeSequence: 1} which restricts physical speed button
+                // to low/high. Changing to {fanModeSequence: 0} allows low/med/high speeds.
+                device.endpoints[0].write('hvacFanCtrl', {fanModeSequence: 0});
```

Comment:
Please put this in the `configure` instead.

Reply:
Thanks for the feedback, @Koenkk - now moved into `configure()`.